dome9/common/resourcetype/resource_type.go

```diff
@@ -9,6 +9,7 @@ const (
 	CloudAccountKubernetes                       = "dome9_cloudaccount_kubernetes"
```

Comment:
After updating the SDK please update the go.mod file to use the newest SDK version